boards.txt

```diff
@@ -28,6 +28,9 @@ arduino_101.build.variant_system_lib=arc32drv_arduino101
 arduino_101.build.vid=0x8087
 arduino_101.build.pid=0x0AB6
 
+arduino_101.bootloader.tool=arduino101load
+arduino_101.bootloader.file=""
```

Comment:
This causes the IDE to give a warning:
`Bootloader file specified but missing: /home/babaltz/.arduino15/packages/Intel-Test/hardware/arc32/0.0.49/bootloaders/""`